python/semantic_kernel/orchestration/kernel_function.py

```diff
@@ -307,7 +307,7 @@ def describe(self) -> FunctionView:
             parameters=self._parameters,
         )
 
-    def __call__(
+    async def __call__(
```

Comment:
I'm not sure if __call__ is allowed to be overwritten as async func...

Reply:
alternative would be to just delete it (there is also no equivalent in dotnet)

Reply:
It works well to override as an async call. One can then `await some_func()`. The code has been updated from the previous sync override to async and notebooks, examples, and tests pass.